mongo-odbc-downloads_template.json

```diff
@@ -0,0 +1,19 @@
+{
+  "manual_link": "https://www.mongodb.com/docs/atlas/data-federation/query/query-with-sql/",
+  "release_notes_link": "https://www.mongodb.com/docs/atlas/data-federation/query/sql/release-notes/",
+  "versions": [
+    {
+      "version": "{RELEASE_VERSION}",
+      "platform": [
+        {
+          "name": "Windows",
+          "download_link": "https://translators-connectors-releases.s3.us-east-1.amazonaws.com/mongosql-odbc-driver/windows/{RELEASE_VERSION}/pkg/mongoodbc.msi"
```

Comment:
Let's change "pkg" to "release"

Reply:
@pmeredit 
It's not yet uploading to "release" from what I can see on master.
Do you know which PR is making that change?